apps/desktop-wallet/src/pages/UnlockedWallet/TransfersPage/FiltersPanel.tsx

```diff
@@ -59,7 +60,8 @@ const FiltersPanel = ({
   const { t } = useTranslation()
   const addresses = useAppSelector(selectAllAddresses)
   const selectAddressesTokens = useMemo(makeSelectAddressesTokens, [])
-  const assets = useAppSelector(selectAddressesTokens)
+  const addressTokens = useAppSelector(selectAddressesTokens)
+  const assets = useSortTokensByWorth(addressTokens)
```

Comment:
Soon enough we'll rename the `assets` everywhere, but for now I keep it to minimize the LOC diff